common/views/components/NewsletterSignup/NewsletterSignup.js

```diff
@@ -100,32 +96,35 @@ class NewsletterSignup extends Component<Props, State> {
   render() {
     const { isConfirmed, isSuccess, isError } = this.props;
     return (
-      <div className='body-text'>
+      <Fragment>
         {isConfirmed &&
-          <Fragment>
-            <h1>Confirmation message</h1>
+          <div className='body-text'>
+            <h1>Thank you for confirming your email address</h1>
+            <p>We’re looking forward to keeping you up-to-date on the topics you’re interested in. You are seeing this page because you clicked on a confirmation link in an email from us, but you can unsubscribe or change your subscription preferences at any time using the link in the emails you receive.</p>
             <p><a href='/whats-on'>Browse our current and upcoming exhibitions and events</a>.</p>
-          </Fragment>
+          </div>
         }
 
         {isSuccess &&
-          <Fragment>
-            <h1>Thank you</h1>
-            <p>Thank you for your interest in receiving updates from Wellcome Collection.</p>
-            <p>If this is first time you have subscribed to a newsletter, you will receive an email asking to confirm your subscription. Please check your email to confirm and start receiving updates.</p>
+          <div className='body-text'>
+            <h1>You’re signed up!</h1>
+            <p>Thank you for signing up to receive updates from us.</p>
+            <p>If this is first time you’ve subscribed to updates from us, you will receive an email asking to confirm your subscription. Please check your email and confirm, so you can start receiving updates.</p>
             <p><a href='/whats-on'>Browse our current and upcoming exhibitions and events</a>.</p>
-          </Fragment>
+          </div>
         }
 
         {isError &&
-          <Fragment>
-            <h1>There was a problem</h1>
+          <div className='body-text'>
+            <h1>Sorry, there&rsquo;s been a problem</h1>
```

Comment:
Could we just use standard characters?

Reply:
Yep (I thought React was going to complain about html entities needing to be escaped, but this only appears to be true for `'`, and not `’`. Will update